project/eScavenge/CLI.py

```diff
@@ -289,11 +289,35 @@ def answer_question(self, args):
         return "There are no more landmarks!"
     return "Incorrect Answer! The Question Was: \n{}".format(self.game.get_team_landmark(self.current_user).question)
 
+@need_admin
+def edit_penalty_value(self, args):
+    try:
+        penalty_value = self.game.set_point_penalty(int(args[1]))
+    except ValueError:
+        return invalid_param
```

Comment:
Instead of invalid_param, we probably want the usage string here
so maybe something like (haven't tested it, but should be close):
```Python3
except ValueError, IndexError:
  return "Proper format..."
```

Reply:
Then you don't need the next check (if Len(args)) since that check comes too late anyways.